silk/tools/otns_manager.py

```diff
@@ -87,6 +87,16 @@ def __init__(self, server_addr: str, logger: logging.Logger):
     self.stub = visualize_grpc_pb2_grpc.VisualizeGrpcServiceStub(self.channel)
     self.logger = logger
 
+  def send_title(self, title: str):
```

Comment:
```suggestion
  def set_title(self, title: str):
```
be consistent with proto?